feat(role-settings): remove conditional titles for role power level settings

MarcWadai opened the pull request:
issue https://github.com/tchapgouv/tchap-web-v4/issues/1017

MarcWadai committed:
feat(role-settings): remove conditional titles for role power level settings

NicolasBuquet commented on modules/tchap-translations/tchap_translations.json:
On parle plus de "position" plutôt que "géolocalisation" dans Tchap.

MarcWadai replied:
merci, modif faite !

NicolasBuquet commented on linked-dependencies/matrix-react-sdk/src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx:
Es-tu sûr que `eventsLevels[M_BEACON_INFO.altName]` n'est pas utilisé ailleurs ?

MarcWadai replied:
Dans ce cas oui. J'ai listé la liste des eventTypes et on recoit 
- m.beacon_info
- org.matrix.msc3672.beacon_info

Alors que dans le code ce sont bien 2 même valeurs (une stable et l'autre instable)

`export const M_BEACON_INFO = new UnstableValue("m.beacon_info", "org.matrix.msc3672.beacon_info")`

NicolasBuquet reviewed (commented):
LGTM!

MarcWadai committed:
feat(role-settings): remove double beacon value and add auto change for both when its modified

MarcWadai commented:
@NicolasBuquet @yostyle 
The double beacon value is not present in element because its an addition on tchap side to allow user to share there location by default. See https://github.com/tchapgouv/synapse-room-access-rules/pull/3
So at the end, I removed visually the duplicate value and changed both of the values when BEACON_INFO is changed

NicolasBuquet reviewed (commented):
Le principe est ok (vu ensemble).
Le changement d'un powerLevel sur NAME devrait changer le powerLevel ALT_NAME.
Cela ne semble pas être le cas.
Tester en inversant l'ordre des changements pour voir si le backend ne prend en compte que le premier changement.
Si c'est le cas, prévoir d'envoyer 2 requêtes de changement distinctes.
